src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -314,25 +314,18 @@ internal async void SetLoadingDone()
         /// <param name="importStatus"></param>
         internal async void SetImportStatus(ImportStatus importStatus)
         {
-            string importSettingsTitle = string.Empty;
+            string importSettingsTitle = Dynamo.Wpf.Properties.Resources.SplashScreenImportSettings;
             string errorDescription = string.Empty;
 
             switch (importStatus)
             {
                 case ImportStatus.none:
-                    importSettingsTitle = Dynamo.Wpf.Properties.Resources.SplashScreenImportSettings;
                     errorDescription = Dynamo.Wpf.Properties.Resources.ImportPreferencesInfo;
                     break;
                 case ImportStatus.error:
-                    importSettingsTitle = Dynamo.Wpf.Properties.Resources.SplashScreenFailedImportSettings;
```

Comment:
Do we need to remove the corresponding resouces?

Reply:
Good call. Removing it...